frontend/android/app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <string name="app_name">MoneyShare</string>
 
-    <string name="base_url_value">http://192.168.99.100:8081/</string>
+    <string name="base_url_value">http://remote.alerant.hu/</string>
```

Comment:
How is this change relevant to the App Logo issue? Maybe create a separate correction commit for this?